src/main/scala/org/broadinstitute/monster/encode/extraction/ExtractionPipelineBuilder.scala

```diff
@@ -144,4 +151,93 @@ object ExtractionPipelineBuilder extends PipelineBuilder[Args] {
     )
     ()
   }
+
+  /**
+    * Pipeline stage which maps batches of query params to output payloads
+    * by sending the query params to the ENCODE search API.
+    *
+    * @param encodeEntity the type of ENCODE entity the stage should query
+    */
+  class EncodeLookup(encodeEntity: EncodeEntity)
```

Comment:
I think this could be a `def` instead of a `class` (nested classes can sometimes wreak havoc in JVM land)